network/peers/connections/conn_handler.go

```diff
@@ -71,13 +71,14 @@ func (ch *connHandler) Handle(logger *zap.Logger) *libp2pnetwork.NotifyBundle {
 		delete(ongoingHandshakes, pid)
 	}
 
+	var ignoredConnection = errors.New("ignored connection")
```

Comment:
convention is to use `ErrIgnoredConnection`. this way from outside you could check if the returned err is `connections.ErrIgnoredConnection`. currently its not clear its err from the name and its not exported.

Reply:
@moshe-blox though if this is used only internal to the package (like I can see now..) then its doesn't matter much..

Reply:
@y0sher it'd be nice to return a bool to signify that connection was accepted, but yeah this func is internal so doesn't matter much